src/fides/api/ops/api/v1/endpoints/consent_request_endpoints.py

```diff
@@ -216,6 +225,86 @@ def get_consent_preferences(
     return _prepare_consent_preferences(db, identity)
 
 
+def load_executable_consent_options(file_path: str) -> List[str]:
+    """Load customer's consentOptions from the config.json file and filter to return only a list
+    of executable consent options"""
+    with open(file_path, encoding="utf-8") as privacy_center_config_file:
+        privacy_center_config: Dict = json.load(privacy_center_config_file)
+        consent_options: List = privacy_center_config.get("consent", {}).get(
+            "consentOptions", []
+        )
+
+        executable_consent_options: List[str] = []
+        for consent in consent_options:
+            data_use: str = consent.get("fidesDataUseKey")
+            if not data_use:
+                continue
+
+            if consent.get("executable"):
+                executable_consent_options.append(data_use)
+            else:
+                logger.info("Consent option: '{}' is not executable.", data_use)
+
+        return executable_consent_options
+
+
+def queue_privacy_request_to_propagate_consent(
+    db: Session,
+    provided_identity: ProvidedIdentity,
+    policy: Union[FidesOpsKey, str],
+    consent_preferences: ConsentPreferences,
+) -> Optional[BulkPostPrivacyRequests]:
+    """
+    Queue a privacy request to carry out propagating consent preferences server-side to third-party systems.
+
+    Only propagate consent preferences which are considered "executable" by the current system. If none of the
+    consent preferences are executable, no Privacy Request is queued.
+    """
+    identity = Identity()
+    setattr(
+        identity,
+        provided_identity.field_name.value,  # type:ignore[attr-defined]
+        provided_identity.encrypted_value["value"],  # type:ignore[index]
+    )  # Pull the information on the ProvidedIdentity for the ConsentRequest to pass along to create a PrivacyRequest
```

Comment:
Note that a separate `ProvidedIdentity` is created for the PrivacyRequest for now